AlphaWallet/Core/Types/RealmConfiguration.swift

```diff
@@ -0,0 +1,60 @@
+// Copyright SIX DAY LLC. All rights reserved.
+
+import Foundation
+import RealmSwift
+
+struct RealmConfiguration {
+    private static let walletsFolderForTests = "wallets"
+    static func configuration(for account: Wallet, server: RPCServer) -> Realm.Configuration {
+        var config = realmConfiguration()
+        config.fileURL = defaultRealmFolderUrl.appendingPathComponent("\(account.address.eip55String.lowercased())-\(server.chainID).realm")
+        return config
+    }
+
+    static var defaultRealmFolderUrl: URL {
+        return realmConfiguration().fileURL!.deletingLastPathComponent()
+    }
+
+    private static func realmConfiguration() -> Realm.Configuration {
+        if isRunningTests() {
+            return Realm.Configuration(fileURL: URL(fileURLWithPath: try! _RLMRealmPathInCacheFolderForFile("default.realm"), isDirectory: false))
+        } else {
+            return Realm.Configuration()
+        }
+    }
+    
+    private static func _RLMRealmPathInCacheFolderForFile(_ fileName: String) throws -> String {
+        let fileManager = FileManager.default
+
+        func createSubDirectoryIfNotExists(name: String) throws -> URL {
+            let documentsURL = fileManager.urls(for: .cachesDirectory, in: .userDomainMask)[0]
+
+            let directory = documentsURL.appendingPathComponent(name)
+            guard !fileManager.fileExists(atPath: directory.absoluteString) else { return directory }
+            try fileManager.createDirectory(atPath: directory.path, withIntermediateDirectories: true, attributes: nil)
+
+            return directory
+        }
+
+        var directory = try createSubDirectoryIfNotExists(name: walletsFolderForTests)
+        directory.appendPathComponent(fileName)
+
+        return directory.path
+    }
+
+    static func removeWalletsFolderForTests(name: String = walletsFolderForTests) throws {
+        let fileManager = FileManager.default
```

Comment:
Can you add as the first line of this function to guard for `isRunningTests()` just to be defensive?